events/templates/events/list_view.html

```diff
@@ -0,0 +1,64 @@
+{% extends "base.html" %}
+
+{% block content %}
+{% load myfilters %}
+<div class = "bg-gray-800">
+  <h2 class="font-bold font-sans text-4xl space-y-6 text-white">
+    Events
+  </h2>
+  <div class = "flex justify-end space-x-3">
+    <a class="text-sm font-bold text-white object-left " href= "{% url 'create_event'%}">Create Event</a>
+    <a class="text-sm font-bold text-white object-left" href= "{% url 'list_event'%}">All events</a>
+
+  </div>
+</div>
+<form method = "GET" novalidate>
+<div class="grid grid-flow-col space-x-3">
+<div class = "bg-gray-800 p-4">
+
+</div> 
+
+  <div>
+    <div class="grid grid-cols-3 grid-rows-3 gap-x-3.5">
+            {% for event in events %}
+            <div class="rounded-lg bg-gray-100 focus-within:ring-2 focus-within:ring-offset-2 focus-within:ring-offset-gray-100 focus-within:ring-indigo-500 overflow-hidden">
+                    {% if event.image %}
+                        <img src="{{event.image.url}}" class = "object-cover pointer-events-none group-hover:opacity-75">
+                    {% else %}
+                        <img src="{{event.image.url}}" class = "object-cover pointer-events-none group-hover:opacity-75"style="max-width: 10%"> 
```

Comment:
Will this work? try adding event without image or remove image and load this page